next/components/forms/info-components/Alert.tsx

```diff
@@ -51,70 +78,60 @@ const Alert = ({
   }
 
   const alertContainer = cx(
-    'flex justify-between rounded-lg lg:px-5 px-3',
+    'flex flex-col items-start gap-2 rounded-lg lg:px-5 px-3 lg:py-4 py-3',
     className,
     {
-      'text-gray-800 flex-col lg:py-4 py-3': variant === 'message',
       'bg-negative-100 text-negative-700': type === 'error' && !solid,
       'bg-success-50 text-success-700': type === 'success' && !solid,
       'bg-gray-100 text-gray-700': type === 'info' && !solid,
       'bg-warning-50 text-warning-700': type === 'warning' && !solid,
 
-      'lg:py-4 p-3 items-center': variant === 'basic',
       'bg-negative-700 text-white': type === 'error' && solid,
       'bg-success-700 text-white': type === 'success' && solid,
       'bg-gray-700 text-white': type === 'info' && solid,
       'bg-warning-700 text-white': type === 'warning' && solid,
     },
-    { 'w-fit max-w-none': fullWidth },
+    { 'w-full max-w-none': fullWidth },
     { 'w-full max-w-[480px]': !fullWidth },
   )
 
-  const contentStyle = cx('w-full', {
-    'text-16': variant === 'basic',
-    'text-16-semibold': variant === 'message',
-    'text-gray-0': solid,
+  const contentStyle = cx('', {
+    'text-16-semibold': title,
+    'text-16': !title,
+    'text-white': solid,
     'text-gray-700': !solid,
   })
 
-  const extraButtonStyle = cx('text-16-medium underline underline-offset-4')
-
-  return variant === 'basic' ? (
+  return (
     <div className={alertContainer}>
-      <div className="flex items-center gap-[14px]">
-        <span className="flex min-w-[22px] justify-center">{icons[type]}</span>
-        <div className={contentStyle}>{message}</div>
-      </div>
-      {close && (
-        <span className="flex h-6 w-6 items-center justify-center">
-          <CloseIcon onClick={close} className="w-6 h-6" />
-        </span>
-      )}
-    </div>
-  ) : (
-    <div className={alertContainer}>
-      <div className="flex flex-row items-center gap-[14px]">
-        <span className="flex min-w-[22px] justify-center">{icons[type]}</span>
-        <div className={contentStyle}>{message}</div>
-      </div>
-      <div
-        className={cx('text-p2 mt-2 w-full pl-9 font-normal', {
-          'text-gray-0': solid,
-          'text-gray-700': !solid,
-        })}
-      >
-        {content}
+      <div className="w-full flex justify-between">
+        <div className="flex gap-[14px]">
+          <span className="flex min-w-[22px] justify-center">{icons[type]}</span>
+          <div className={contentStyle}>{title || message}</div>
+        </div>
+        {buttonsPosition === 'right' && (
+          <AlertButtons className="hidden lg:flex pl-3" buttons={buttons} />
+        )}
```

Comment:
let's remove this variant/configuration altogether - if there are to be buttons, they will be at the bottom